samples/cpp/text_generation/README.md

```diff
@@ -36,9 +60,72 @@ ov::genai::LLMPipeline pipe(model_str, weights_tensor, tokenizer, device);
 ```
 For the sake of brevity the code above does not include Tokenizer decryption. For more details look to encrypted_model_causal_lm sample.
 
-### Troubleshooting
+## Sample Descriptions
+
+### 1. Text Generation (`text_generation`)
+- **Description:** Basic text generation using a causal language model.
+- **Main Feature:** Demonstrates simple text continuation.
+- **Run Command:**
+  ```bash
+  ./text_generation -m <model> -i "Hello, how are you?" -d CPU
```

Comment:
1. If it's an example command, it must be specific, replace `<model>` with TinyLlama. If it's a help message, align it with what the sample would print.
2. There's no `text_generation` target.
3. The samples don't take device as an arg. They also don't have named args

Reply:
done